src/Collector/WebViewCollector.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Debug\Collector;
+
+use Yiisoft\View\Event\WebView\AfterRender;
+
+final class WebViewCollector implements CollectorInterface
+{
+    use CollectorTrait;
+
+    private array $view = [];
```

Comment:
(Naming) Not obvious that view is an array.

Reply:
What about `$renders`?

Reply:
Same problem - ambiguity. `$renders` can be imagined as a boolean for example.

Reply:
use `webViewCollector`, mnemonically it tells you that it is collecting the data from the `WebView::class` .